project/Dependencies.scala

```diff
@@ -25,24 +25,26 @@ object Dependencies {
   val akkaSlf4j = akkaGroup %% "akka-slf4j" % akkaVersion
 
   val scala_2_1X = "2\\.1([0-9])\\.[0-9]+.*".r
-  val spark_1_X = "[a-zA-Z]*1\\.([0-9]+)\\.([0-9]+).*".r
+  val spark_X_Y = "[a-zA-Z]*([0-9]+)\\.([0-9]+)\\.([0-9]+).*".r
   val extractVs = "[a-zA-Z]*(\\d+)\\.(\\d+)\\.(\\d+).*".r
 
-  val defaultSparkVersion = sys.props.getOrElse("spark.version", "1.6.0")
+  val defaultSparkVersion = sys.props.getOrElse("spark.version", "1.6.1")
```

Comment:
P.S. word `default` looks misleading here. It's the **actual** `spark.version` to use
(not to be fixed in this PR ;] ).